src/hmrc-design-patterns/eori-numbers/index.njk

```diff
@@ -49,7 +49,14 @@ layout: twoColumnPage.njk
   })
 }}
 
-<p class="govuk-body">Use a single text input and allow the user to enter ‘GB’ or ‘XI’ and 12 to 15 numbers. This can be with or without spaces, and using uppercase or lowercase letters, for example, GB123456123456 or XI123456123456.</p>
+<p class="govuk-body">Use a single text input and allow the user to enter the EORI type (for example ‘GB’ or ‘XI’) and numbers. This can be with or without spaces, and using uppercase or lowercase letters.</p>
```

Comment:
Could you put a comma after the for example please: '(for example, 'GB' or 'XI')'
Otherwise OK.